downloaded/installLisk.sh

```diff
@@ -303,6 +320,7 @@ parse_option() {
 			u) URL="$OPTARG" ;;
 			0) SYNC="$OPTARG" ;;
 			c) CLEAN_DB="yes" ;;
+			s) LISK_VERSION_NUMBER="$OPTARG" ;;
```

Comment:
Why `-s`? (and not, say, `-v`?)

Reply:
I was saving `-v` for either a verbose mode or having `installLisk.sh -v` spit out its own version number, if we go down that road